pkg/loader/fileloader.go

```diff
@@ -205,13 +217,42 @@ func isPathEqualToOrAbove(path string, roots []string) error {
 	return nil
 }
 
-// Load returns content of file at the given relative path.
+// Load returns content of file at the given relative path,
+// else an error.  The path must refer to a file in or
+// below the current Root().
 func (l *fileLoader) Load(path string) ([]byte, error) {
 	if filepath.IsAbs(path) {
-		return nil, fmt.Errorf(
-			"must use relative path; '%s' is absolute", path)
+		return nil, l.loadOutOfBounds(path)
```

Comment:
The previous error message looks better. We can leave it unchanged.

Reply:
Message updated.

This is now a change from
```
must use relative path; '/etc/tls.key' is absolute.
```
to
```
"security; file `/etc/tls.key` is not in or below `/the/kustomization/root`,
```
How's that?